Taskfile.yml

```diff
@@ -232,6 +234,10 @@ tasks:
         cd client
         PATH="{{.G_NODEJS_22_BIN_DIR}}":$PATH npm run build -- \
         --output-path "{{.OUTPUT_DIR}}/client"
+      - |-
```

Comment:
Shall we add a step to checkout the `yscope-log-viewer` submodule?

Reply:
Added task `log-viewer-webui-submodules`